lib/src/style/QlementineStyle.cpp

```diff
@@ -3007,15 +3005,6 @@ QRect QlementineStyle::subControlRect(ComplexControl cc, const QStyleOptionCompl
 			if (const auto* scrollBarOpt = qstyleoption_cast<const QStyleOptionSlider*>(opt)) {
 				const auto horizontal = scrollBarOpt->orientation == Qt::Horizontal;
 				const auto& rect = scrollBarOpt->rect;
-				auto position = static_cast<qreal>(scrollBarOpt->sliderPosition);
```

Comment:
Why did you removed these lines?

Reply:
`position` was never used so they had no effect (is this normal?)